packages/nova-react/src/graphql/nova-graphql-provider.test.tsx

```diff
@@ -1,42 +1,32 @@
-/**
- * @jest-environment jsdom
- */
-
 import * as React from "react";
 import type { NovaGraphQL } from "@nova/types";
-import { render } from "@testing-library/react";
+import { render } from "vitest-browser-react";
+import { describe, it, expect, vi } from "vitest";
 import { NovaGraphQLProvider, useNovaGraphQL } from "./nova-graphql-provider";
 
 describe(useNovaGraphQL, () => {
   it("throws without a provider", () => {
-    expect.assertions(1);
```

Comment:
I noticed we left these in some of the other tests. Any specific reason why they're removed here?

Reply:
Will check carefully for these and bring back - agent didn't like them for some reason